src/FactionsPro/FactionListener.php

```diff
@@ -97,8 +97,10 @@ public function factionPVP(EntityDamageEvent $factionDamage) {
 		}
 	}
 	public function factionBlockBreakProtect(BlockBreakEvent $event) {
-		if($this->plugin->isInPlot($event->getPlayer())) {
-			if($this->plugin->inOwnPlot($event->getPlayer())) {
+       $x = $event->getBlock()->getX();
+       $z = $event->getBlock()->getZ();
+		if($this->plugin->pointIsInPlot($x, $z)) {
+			if($this->plugin->factionFromPoint($x, $z) == $this->plugin->getFaction($event->getPlayer()) {

```

Comment:
Did you mean parentheses instead of bracket? I found a missing parentheses, but brackets seem correct to me.